src/components/Tool/FixCommitmentInfo/FixCommitmentInfo.tsx

```diff
@@ -9,12 +9,12 @@ import {
   CircularProgress,
 } from '@material-ui/core';
 import { Trans, useTranslation } from 'react-i18next';
-import { useAccountListId } from '../../../../src/hooks/useAccountListId';
 import { useGetInvalidStatusesQuery } from './GetInvalidStatuses.generated';
 import Contact from './Contact';
 import NoContacts from './NoContacts';
 import { contactTags } from './InputOptions/ContactTags';
 import { frequencies } from './InputOptions/Frequencies';
+import theme from 'src/theme';
```

Comment:
Hmm, this import doesn't look right..

Reply:
I think I've had this happen before. Something that happened after I updated vscode, changed my default import settings.